org-formation/organization.yaml

```diff
@@ -288,6 +289,20 @@ Organization:
         budget-alarm-threshold: 2000
         budget-alarm-threshold-email-recipient: aws-iatlas-prod@sagebase.org
 
+  # Provisioner repository: https://github.com/ncihtan/aws-htan-infra
+  HtanDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: htan-dev
+      RootEmail: aws-htan-dev@sagebase.org
+      Alias: org-sagebase-htan-dev
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        Department: CompOnc
+        Project: htan
+        budget-alarm-threshold: 10000
+        budget-alarm-threshold-email-recipient: aws-htan-dev@sagebase.org
```

Comment:
have you created this google group already?

Reply:
Yup! 

https://groups.google.com/a/sagebase.org/g/aws-htan-dev